### PR TITLE
[v7r1] ComponentInstaller uses --cfg

### PR DESCRIPTION
**Pull request by chaen**
BEGINRELEASENOTES

*Framework
CHANGE: ComponentInstaller uses --cfg
*Tests
CHANGE: uses --cfg

ENDRELEASENOTES

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 name: dirac-development
```
Comment (fstagni) [inline environment.yml:1]: Why there are changes in this file?

Comment (chaen) [reply environment.yml:1]: Probably a poor rebase...

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -38,6 +39,10 @@ jobs:
             CLIENT_USE_M2CRYPTO: Yes
           - SERVER_USE_M2CRYPTO: No
             CLIENT_USE_M2CRYPTO: No
+          # Fail deprecation
```
Comment (chrisburr) [inline .github/workflows/integration.yml:42]: To be consistent with the other options:
```suggestion
          ###### Fail deprecation
```

Comment (chaen) [reply .github/workflows/integration.yml]: done

**Comment by chaen** [comment]
Unfortunately, there is no possibility at the moment to have a job allowed to fail, while having the green arrow for the tests. So I propose to split this PR, and we will merge the CI part once this is resolved https://github.com/actions/runner/issues/2347

**Comment by chaen** [comment]
@chrisburr any idea why all the PRs have issues starting all the tests lately ? Do we reach a quota or something ?

**Comment by chrisburr** [comment]
There isn't a quota for public projects.

I think it's just instability on the GitHub side with an API call getting lost somewhere. The uptime shown on [their status page](https://www.githubstatus.com/) isn't great and has been getting worse over the last few months. I've opened a support ticket about it.

**Comment by chaen** [comment]
Yeah I did check the status page but since it was green for the days we had issues, I assumed it was something else...

**Comment by chrisburr** [comment]
I got an impressively quick response:

> Ahh sorry about that -- this was a recent issue that our engineers just shipped a fix for a few hours ago. The symptoms match the runs you shared where the run reports as completed (in the UI under the Actions tab and in the API), but it has jobs reporting as queued.
> I see the most recent example you shared is from ~15 hours ago, so before the fix went out. Can you let us know if you see any new runs that demonstrate this behavior?

And indeed, the [actions overview](https://github.com/DIRACGrid/DIRAC/actions?query=workflow%3A%22Basic+tests%22+branch%3Arel-v7r1_FIX_cfgArg) shows the run as completed even if the detailed page doesn't. If anyone sees it happen again let me know and I'll continue the thread.

**Comment by fstagni** [comment]
The comment in the op should be adapted. Anyway the main question is: do you still need to add something?

**Comment by chaen** [comment]
Ready to go

**Comment by fstagni** [comment]
There are a few more places in the tests where `--cfg` can be added, e.g.:
- https://github.com/DIRACGrid/DIRAC/blob/integration/tests/Jenkins/utilities.sh#L401
- In this function: https://github.com/DIRACGrid/DIRAC/blob/integration/tests/Jenkins/utilities.sh#L1021
- In Test_TimeLeft.sh

Would you mind adding it there too?

**Comment by chaen** [comment]
No problem.
For https://github.com/DIRACGrid/DIRAC/blob/integration/tests/Jenkins/utilities.sh#L401 it's a bit strange though that the configuration file is is passed twice

**Comment by fstagni** [comment]
> No problem.
> For https://github.com/DIRACGrid/DIRAC/blob/integration/tests/Jenkins/utilities.sh#L401 it's a bit strange though that the configuration file is is passed twice

The second (with "-F" is the file to update.

**Pull request merged**